client/src/pages/Home/index.js

```diff
@@ -1,24 +1,65 @@
-import { useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
 
 import { Pagination, ResourceList } from "../../components";
 import { useSearchParams } from "../../hooks";
 import { ResourceService, useService } from "../../services";
 
-export function Home() {
+const Home = () => {
 	const resourceService = useService(ResourceService);
 	const searchParams = useSearchParams();
-	const [{ lastPage, resources } = {}, setEnvelope] = useState();
+	const [{ lastPage, resources } = {}, setEnvelope] = useState({});
```

Comment:
❓ Why was this change introduced? Had you experienced some issue with the previous implementation?